lib/services/local/users.go

```diff
@@ -1491,6 +1491,20 @@ func (s *IdentityService) GetKeyAttestationData(ctx context.Context, publicKey c
 	return &resp, nil
 }
 
+func fingerprintSha256(pubDER []byte) string {
+	uuid := uuid.NewHash(crypto.SHA256.New(), uuid.UUID{}, pubDER, 0)
+	return uuid.String()
+}
+
+// DELETE IN 13.0.0
+func oldFingerprintSha256(pub crypto.PublicKey) (string, error) {
```

Comment:
```suggestion
func keyAttestationDataFingerprintV11(pub crypto.PublicKey) (string, error) {
```

oldKeyAttestationBlabla also works, but this way we add more info to the name.